Doc/whatsnew/3.7.rst

```diff
@@ -330,10 +330,9 @@ that may require changes to your code.
 Changes in the Python API
 -------------------------
 
-* :meth:`pkgutil.walk_packages` now raises ValueError/TypeError if path is
-  string/bytes object.
-  Previously an empty list was returned. (Contributed by Sanyam Khurana in
-  :issue:`24744`.)
+* :meth:`pkgutil.walk_packages` now raises ValueError if path is a string
+  object. Previously an empty list was returned. (Contributed by Sanyam
+  Khurana in :issue:`24744`.)
```

Comment:
'object' is redundant here.  Also, mark up "path" as an argument by surrounding it with asterisks.